src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -997,6 +1000,143 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
 
+        [Fact]
+        public async Task PostAsync_ResponseContentRead_RequestContentDisposedAfterResponseBuffered()
```

Comment:
ok. Thx.  But it's not obvious when quickly looking at the test collateral that we have tests covering that. As we add tests, we need to figure out a way to catalog them so that we know we have coverage across the matrix of possibilities. Right now it is ad-hoc and we have to visually scan the test sources which contain to grow.